packages/katello/katello/hostname-change.rb

```diff
@@ -167,6 +167,28 @@ def using_custom_certs?(scenario_answers)
       scenario_answers["certs"]["server_cert_req"]
     end
 
+    def delete_puppet_certs
+      puppet_ssldir = @scenario_answers['puppet']['ssldir']
+      files = [
+        'ca/ca_crt.pem',
+        'ca/ca_crl.pem',
+        'ca/infra_crl.pem',
+        'certs/ca.pem',
+        'crl.pem',
+        'ca/ca_pub.pem',
+        'ca/inventory.txt',
+        'ca/infra_inventory.txt',
+        'ca/infra_serials',
+        'ca/serial',
+        'ca/root_key.pem',
+        'ca/ca_key.pem'
+      ]
+
+      files.each do |file|
+        run_cmd("rm -rf #{File.join(puppet_ssldir, file)}")
```

Comment:
Wouldn't it be cleaner to use native Ruby functions? Or at least quote the argument.

Reply:
I think so, but run_cmd is consistent with everything else in the script and apparently it does some cleanup and handling of any errors